pkgs/by-name/lu/lunacy/package.nix

```
@@ -0,0 +1,100 @@
+{ stdenv
+, lib
+, fetchurl
+, dpkg
+, makeWrapper
+, autoPatchelfHook
+, zlib
+, libgcc
+, fontconfig
+, libX11
+, lttng-ust
+, icu
+, libICE
+, libSM
+, libXcursor
+, openssl }:
+
+stdenv.mkDerivation rec {
```

Avoid using `rec` and instead convert this to be a function. See https://discourse.nixos.org/t/avoid-rec-expresions-in-nixpkgs/8293
```suggestion
stdenv.mkDerivation (finalAttrs: {
```